### PR TITLE
Update stewards to triagers

### PR DESCRIPTION
**Pull request by damianavila**
A naming follow-up of our improved support process recognizing the "triager" role even more.

### CODE REVIEW

File: projects/managed-hubs/support.md [hunk 1]
```diff
@@ -111,9 +111,9 @@ Hub Administrators
 
 ### Who are our Support Team?
 
-The Support Steward role rotates through the below Support Team members.
+The Support Triager role rotates through the below Support Team members.
 
-```{include} ../../_data/tmp/support-stewards.txt
+```{include} ../../_data/tmp/support-triagers.txt
```
Comment (damianavila) [inline projects/managed-hubs/support.md:116]:
```suggestion
```{include} ../../_data/tmp/support-stewards.txt
```

Comment (damianavila) [reply projects/managed-hubs/support.md:116]: Do not change the name of the file for now...

**Comment by damianavila** [comment]
This is not changing the names of scripts (or groups in Slack) because that will be a bigger change (no time for that right now) but it is an improvement from the previous state (and we can complete it later down the road).

**Review by yuvipanda — APPROVED** [review]

**Comment by damianavila** [comment]
Related: https://github.com/2i2c-org/team-roles-geekbot-sweep/pull/200
And I might need to update this one after that one is merged!

**Comment by choldgraf** [comment]
@damianavila is this one ready to merge? Or do you need to take action per https://github.com/2i2c-org/team-compass/pull/769#issuecomment-1739875798 ?

**Comment by damianavila** [comment]
> @damianavila is this one ready to merge? Or do you need to take action per https://github.com/2i2c-org/team-compass/pull/769#issuecomment-1739875798 ?

I need to take an action, IIRC. I will update it soon.

**Comment by yuvipanda** [comment]
@damianavila can you resolve the conflict and get this through? Or shall I?

**Comment by damianavila** [comment]
Resolved conflict and made some updates after https://github.com/2i2c-org/team-roles-geekbot-sweep/pull/200.

**Comment by damianavila** [comment]
@sgibson91, I made some minimal invasive changes in my last commit to the `gen_support_steward.py` file.
When I check the built site on this PR, I can see this warning message: Slack data did not download
See, https://2i2c-team-compass--769.org.readthedocs.build/projects/managed-hubs/support/#who-are-our-support-team
Funny thing, it seems to be silently broken on the current site: https://compass.2i2c.org/projects/managed-hubs/support/#who-are-our-support-team.
Any thoughts given that you are the bot master here 😉 ?

**Comment by sgibson91** [comment]
@damianavila from the readthedocs build logs:

```
Generating support stewards gallery encountered an error:
SLACK_BOT_TOKEN does not exist. Skipping Support Steward table generation...
```

**Comment by sgibson91** [comment]
It does though...

Screenshot 2024-01-30 at 15 00 16

**Comment by sgibson91** [comment]
From the docs: https://docs.readthedocs.io/en/stable/environment-variables.html#environment-variables-and-build-process

> Pull Request builds
> Custom environment variables that are not marked as Public will not be available in [pull request builds](https://docs.readthedocs.io/en/stable/pull-requests.html)

Ours in not public, so I guess we can expect it to fail in PR builds

**Comment by sgibson91** [comment]
From the last build for `latest` version of the docs:

```
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/2i2c-team-compass/checkouts/latest/_data/support_stewards/gen_support_stewards.py", line 157, in <module>
    main()
  File "/home/docs/checkouts/readthedocs.org/user_builds/2i2c-team-compass/checkouts/latest/_data/support_stewards/gen_support_stewards.py", line 117, in main
    usernames_and_avatars = slack.get_users_in_usergroup("support-stewards")
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/docs/checkouts/readthedocs.org/user_builds/2i2c-team-compass/checkouts/latest/_data/support_stewards/gen_support_stewards.py", line 87, in get_users_in_usergroup
    self._get_user_ids(usergroup_name)
  File "/home/docs/checkouts/readthedocs.org/user_builds/2i2c-team-compass/checkouts/latest/_data/support_stewards/gen_support_stewards.py", line 43, in _get_user_ids
    self._get_usergroup_id(usergroup_name)
  File "/home/docs/checkouts/readthedocs.org/user_builds/2i2c-team-compass/checkouts/latest/_data/support_stewards/gen_support_stewards.py", line 31, in _get_usergroup_id
    index = next(
            ^^^^^
StopIteration
making output directory... done
matplotlib is not installed, social cards will not be generated
```

1. The Slack group has been renamed already, so merging this PR should fix it
2. There seems to be a complaint about matplotlib not being installed as well, so may need to do that too

**Comment by sgibson91** [comment]
May have to do a more robust sweep of gen-support-stewards.py

**Comment by damianavila** [comment]
Following @sgibson91 recommendation, let's merge this one now!

**Comment by damianavila** [comment]
Actually worked: https://compass.2i2c.org/projects/managed-hubs/support/#who-are-our-support-team
Thanks for your feedback @sgibson91!

**Pull request merged**